data-science/jupyterlab-pyspark-tensorflow/Dockerfile

```
@@ -1,15 +1,7 @@
-ARG JUPYTERLAB_VERSION
+ARG H2OFLOW_VERSION
 ARG PYSPARK_VERSION
 
-FROM ydata/jupyterlab-${JUPYTERLAB_VERSION}_pyspark-${PYSPARK_VERSION}
-
-USER root
-
-RUN apt-get update && apt-get install -yq --no-install-recommends openjdk-8-jre \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
-
-USER ydata
+FROM ydata/h2oflow-${H2OFLOW_VERSION}_pyspark-${PYSPARK_VERSION}
```

```suggestion
FROM ydata/h2oflow-${H2OFLOW_VERSION}_pyspark-${PYSPARK_VERSION}
```
please revert to avoid breaking the jupyter notebook